tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/profiler/ProfilerTest.java

```diff
@@ -41,8 +41,9 @@
 public class ProfilerTest extends BaseTest {
     private static final String testFileLocation = Paths.get("src/test/resources/profiler")
```

Comment:
Shall we use here Paths.get(src, test, resources,profiler). test will not work in windows